draft-liu-nasr-architecture.md

```diff
@@ -101,38 +101,76 @@ This process is repeated periodically to continuously assure compliance.
-|   | Attester  |   |          | | Attester      | |     |  | Attester     | |           |   | Attester  |   |
-|   |           +---+----------+>| Vendor A      +-+-----+->| Vendor B     +-+-----------+-->|           |   |
-|   +-----------+   |Update PE | +---------------+ |     |  +--------------+ | Update PE |   +-----------+   |
-|                   |  with    |                   |     |                   |   with    |                   |
-|                   |AR/RE/PoT |  Operator 1       |     |   Operator 2      | AR/RE/PoT |                   |
-+-------------------+          +-------------------+     +-------------------+           +-------------------+
-                                                                                                              
-
-In a more generalized scenario where multiple operators, having devices from different vendors, collaborate to provide a secure leased line service. 
-
+                          +----------------------------------------------------------------+                         
+                          |                                                                |                         
+                          |       +------------+                 +------------+            |                         
+                          |       | Verifier   |                 | Verifier   |    ...     |                         
+                          |       | Vendor A   |                 | Vendor B   |            |                         
++-------------------+     |       +----^-----+-+                 +---^----+---+   Vendors  |    +-------------------+
+|                   |     |            |     |                       |    |                |    |                   |
+|                   |     +------------+-----+-----------------------+----+----------------+    |                   |
+|    Client  A      |                  |     |                       |    |                     |    Client  B      |
+|                   |  Path            |     |                       |    |         Path        |                   |
+|  +--------------+ |  Request         |     |                       |    |         Attestation |   +------------+  |
+|  |  Relying     +-+----------+ +-----+-----+-------+        +------+----+-------+ Result (PAR)|   |  Relying   |  |
+|  |  Party       | |          | |     |     |       |        |      |    |       |     +-------+---+  Party     |  |
+|  +---+----------+<+--------+ | |     |     |       |  Intra |      |    |       |     |       |   +------------+  |
+|      |            |        | | |  RE |     |AR     |  ISP   |   RE |    | AR    |     |       |           ^       |
+|      |            | Answer | +-> +---+-----v-----+ |  API   |  +---+----v----+  |     |       |           |       |
+|      |Path        | Report |   | | Orchestrator  +-+--------+->| Orchestrator|<-+-----+       |           |       |
```

Comment:
Usually, response is used more than answer, respond report

Reply:
It is actually answer/report, but i get your point. 
I will just leave "report"